tools/tuf/releaser.sh

```diff
@@ -313,6 +319,8 @@ elif [[ $ACTION == "update-timestamp" ]]; then
     push_to_staging
 elif [[ $ACTION == "release-to-production" ]]; then
     release_to_production
+elif [[ $ACTION == "create-fleetd-release-pr" ]]; then
```

Comment:
Do we need this as an `ACTION`? If so, we need to update a couple more places where `$ACTION` is checked.

Reply:
The idea here is to allow you to create the changelog PR after doing the release, if you did `SKIP_PR=1` during the release.  I think it's worthwhile since it involves running a script to generate the changelog.  I'll update the other spot.